sources/packages/backend/libs/sims-db/src/entities/cas-distribution-account.model.ts

```diff
@@ -0,0 +1,57 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+import { OfferingIntensity } from ".";
+
+/**
+ * CAS distribution account information to be reported in invoice details.
+ */
+@Entity({ name: TableNames.CASDistributionAccounts })
+export class CASDistributionAccount extends RecordDataModel {
+  /**
+   * Auto-generated sequential primary key column.
+   */
+  @PrimaryGeneratedColumn()
+  id: number;
+  /**
+   * SIMS award value codes.
+   */
+  @Column({
+    name: "award_value_code",
+  })
+  awardValueCode: string;
+  /**
+   * Offering intensity to allow same awards to potentially have
+   * different distribution accounts for a same award code.
+   */
+  @Column({
+    name: "offering_intensity",
+    enum: OfferingIntensity,
```

Comment:
Don't we require `type: "enum"`?

Reply:
Yes, we do and we have it at least in 55 places. I did not find a reason why the `OfferingIntensity` was not using it in other places where the enum is present.
We had  some issues with `enum` a long time ago and if I am not wrong the understanding was to use it alongside the enum name.
It is also part of [the wiki examples](https://github.com/bcgov/SIMS/wiki/Development---Standards,-Decisions,-and-Troubleshooting#typeorm-database-models).

Found the below AC also in the PR #1036
```
Adding enumName to all properties of entities which reference a DB enum object.
```

Reply:
I added the type to be aligned with other `enum` types but I did not find a reason why only the `OfferingIntnsity` has been declared without other common properties.
```ts
 @Column({
    name: "offering_intensity",
    nullable: false,
  })
  offeringIntensity: OfferingIntensity;
```
For instance, there was a PR in the past to add `enumName` to all the enum and later it was removed only for the `offering_intensity` column. Was it because it is the only one that is shared across multiple tables?

Reply:
Not sure, I dont remember why we  removed it?

Reply:
The plan is to add this new column with the properties that we use for other enums and run some tests to see if some issues arise.